Lib/glyphsLib/builder/font.py

```diff
@@ -69,27 +70,58 @@ def to_ufo_font_attributes(self, family_name):
 
 
 INFO_FIELDS = (
-    ("unitsPerEm", "upm", True),
-    ("versionMajor", "versionMajor", True),
-    ("versionMinor", "versionMinor", True),
-    ("copyright", "copyright", False),
-    ("openTypeNameDesigner", "designer", False),
-    ("openTypeNameDesignerURL", "designerURL", False),
-    ("openTypeNameManufacturer", "manufacturer", False),
-    ("openTypeNameManufacturerURL", "manufacturerURL", False),
+    ("unitsPerEm", "upm"),
+    ("versionMajor", "versionMajor"),
+    ("versionMinor", "versionMinor"),
+)
+
+PROPERTIES_FIELDS = (
+    ("copyright", "copyrights", 0),
+    ("openTypeNameDesigner", "designers", 9),
+    ("openTypeNameDesignerURL", "designerURL", None),
+    ("openTypeNameManufacturer", "manufacturers", 8),
+    ("openTypeNameManufacturerURL", "manufacturerURL", None),
+    ("openTypeNameDescription", "descriptions", 10),
+    ("openTypeNameSampleText", "sampleTexts", 19),
```

Comment:
@khaledhosny I was working on implementing this in fontc and noticed that this is missing `familyNames` which can also be localized. fontc happens to have one of its test fonts with a localized familyNames:

https://github.com/googlefonts/fontc/blob/9925c92ff9a3b253548f849aab0ddbe29735c5ee/resources/testdata/glyphs3/WghtVar.glyphs#L354-L360

Was this intentional to exclude it? Or the list was never meant to be exhaustive (best effort basis) and may be missing other localizable fields in addition to familyNames?